code/game/machinery/suit_storage_unit.dm

```diff
@@ -165,7 +165,48 @@
 		playsound(loc, I.usesound, 100, 1)
 		to_chat(user, text("<span class='notice'>You [panel_open ? "open up" : "close"] the unit's maintenance panel.</span>"))
 		updateUsrDialog()
+		return

```

Comment:
Trailing return

Reply:
Early return, stops the rest of the if statements from being needlessly checked, tiger wants me to keep them in